static/app/components/discover/transactionsTable.tsx

```diff
@@ -238,4 +258,16 @@ const StyledIconQuestion = styled(QuestionTooltip)`
   left: 4px;
 `;
 
+const StyledGridCell = styled('div')`
+  display: grid;
+  grid-template-columns: max-content 1fr;
+  grid-gap: ${space(0.5)};
+  align-items: center;
+`;
+
+const StyledIconPlay = styled(IconPlay)`
+  position: relative;
+  top: -1px;
```

Comment:
With this little tweak the icon gets centered, I believe that due to the font, aligning them with the text with props like `align-items` don't do the trick alone.